### DISCUSSION TITLE
Not able to select text in terminal

### DISCUSSION THREAD

**Discussion by joonshakya** [t3-oss/create-t3-turbo#1041] [Q&A]
I had been using previous version of create-t3-turbo where all the logs would show up in the same place.
I just upgraded to the newest version and now it's cool that logs show up separately but I am not able to select, copy the text in the terminal. Is there any solution to this?

Thank you

**Comment by juliusmarminge** [comment]
Try bumping turbo to latest canary. They've made some improvements to the experimental UI lately

**Reply by joonshakya** [reply]
Ohh thank you so much.
Also one of the other problems I had was,
I was using generated prisma clent as a package in my monorepo so that expo can import things from prisma's generated folder.
It by default has package name ".prisma/client" in package.json inside prisma generated folder
and I used to import enums from ".prisma/client" which used to work in expo
it did give me a warning saying packages cannot have dots and slashes in their names but still it used to work fine

in the newest version, it renames the package name to "prisma-client-some-random-hash-value" which changes on every prisma generate command.
I am using the same version of prisma as my previous repo.
Do you know of a way to solve this?
